### DISCUSSION TITLE
이미지 저장 base64 방법에 관하여

### DISCUSSION THREAD

**Discussion by june0216** [devocean-green-dev/GreenDev_BE#5] [Ideas]
저희가 이미지 저장을 할 때 base64로 인코딩 후 디비에 저장하기로 논의했습니다/ 
하지만 이 경우,DB 차지하는 용량이 많이 늘고, 트래픽도 는다는 단점이 있다고 합니다! 
이에 따라 s3에 파일로 저장하고 파일키만 디비에 저장하는 방법으로 생각중인데 의견부탁드립니다!

**Comment by yymin1022** [comment]
@june0216 
이부분은 더 좋은 방법이 있다면 그렇게 구현해주시면 좋을 것 같아요!
제가 base64를 사용했을 땐, 서버에 파일 자체가 저장되어있고 백엔드가 정해진 경로에서 이미지파일을 찾아 인코딩해서 클라이언트로 보내주는거였어서요,
DB에 들어가면 말씀하신대로 과하게 무거워지는 문제가 있겠네요!
키만 저장하는 쪽이 훨씬 좋을 것 같습니다~

**Reply by june0216** [reply]
아 그렇군요!! 답변 감사합니다~ 좋은 하루 되세요~
